### DISCUSSION TITLE
Simplify Keptn Website Messaging

### DISCUSSION THREAD

**Discussion by ghost** [keptn/keptn#6704] [Ideas]
The keptn is powerful, but with that power comes options and complexity. This discussion is to gather thoughts on how the Keptn website text, imagery, structure and messaging could be improved to simplify the Keptn, **particularly for new users**.

@bradmccoydev we've spoken about this so you may have opinions?

**Comment by bradmccoydev** [comment]
Yes @Dynatrace-Adam-Gardner I think we could do a session with some beginners of Keptn that come from a strong engineering background to identity some improvements we can make in this area, as in my experience Keptn hard to learn out of the box and hopefully this doesn't deter  people away from it when starting out

**Comment by ghost** [comment]
Thanks Brad. I'm happy to facilitate this session.

My 2c here is that the website text and imagery require a rework to get across the newer functionality available in Keptn.

The current content was perfect for when Keptn's sequences were quite opinionated (you didn't used to be able to define your own sequences). Now that Keptn users have the ability to craft any arbitrary sequences of tasks, the website is (IMHO) too geared towards observability and CD usecases. This (naturally) leads to comments like "keptn is a (tool x) replacement".

I'd love messaging that followed the facts:

1. Define the sequence and tasks that you want to achieve
2. Bring your preferred tooling to accomplish each task
3. Ask the Keptn to run your sequence - Keptn will co-ordinate the rest

SREs define the process. Teams bring their preferred tooling. Everyone is happy. No more tool mandates, no more skills gaps, no more duplicated / fragile / buggy code.

Keptn also needs a one sentence "mission statement". Something that a brand new user could read and say "yes, I need some of that".

Some examples that I like:

> Cloud Events: A specification for describing event data in a common way

> Helm: Helm helps you manage Kubernetes applications - Helm Charts help you define, install, and upgrade even the most complex Kubernetes application.

> Falco: Falco, the cloud-native runtime security project, is the de facto Kubernetes threat detection engine

> Litmus: Litmus is an end-to-end chaos engineering platform for cloud native infrastructure and applications.
